lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2013/data/ashrae_90_1_2013.spc_typ.json

```diff
@@ -18955,10 +18955,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
+      "ventilation_primary_space_type": "Office",
+      "ventilation_secondary_space_type": "Buildings Main entry lobbies",
       "ventilation_per_area": 0.06,
-      "ventilation_per_person": null,
+      "ventilation_per_person": 5.0,
```

Comment:
The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.